src/SUMMARY.md

```diff
@@ -2,36 +2,28 @@
 
 [Introduction](introduction.md)
 
-# Getting Started With Stwo-Cairo
+- [Getting Started With Stwo-Cairo](getting-started/index.md)
 
-Clarify that Stwo-Cairo is a "particular application" of Stwo.
+  - [Quickstart](getting-started/quickstart.md)
+  - [Troubleshooting](getting-started/troubleshooting.md)
```

Comment:
Did we get around to contacting Raz to see it it makes sense to move this content over there to avoid duplication?